test/reductions/vass/reductions-perf.perfkeys

```diff
@@ -0,0 +1,9 @@
+success  single reduction 1d:
+success  multiple reductions 1d:
+success  multiple reductions 2d:
+success  multiple reductions 3d:
+n00
```

Comment:
Should n00, n11, n22, n33, and M be perfkeys?  I'm not sure what having them here will do in the collected data and graph.